cmd/common/flags.go

```diff
@@ -3,26 +3,65 @@ package common
 import (
 	"context"
 	"fmt"
+	"strings"
 
 	flag "github.com/spf13/pflag"
 
 	consensus "github.com/oasisprotocol/oasis-core/go/consensus/api"
 )
 
 var (
-	selectedHeight int64
-	force          bool
-	answerYes      bool
+	// HeightFlag is the flag for specifying block height.
+	HeightFlag *flag.FlagSet
+
+	// ForceFlag is a force mode switch.
+	ForceFlag *flag.FlagSet
+
+	// AnswerYesFlag answers yes to all questions.
+	AnswerYesFlag *flag.FlagSet
+
+	// FormatFlag specifies the command's output format (text/json).
+	FormatFlag *flag.FlagSet
 )
 
-// HeightFlag is the flag for specifying block height.
-var HeightFlag *flag.FlagSet
+// FormatType specifies the type of format for output of commands.
+type FormatType string
 
-// ForceFlag is a force mode switch.
-var ForceFlag *flag.FlagSet
+// Supported output formats for the format flag.
+const (
+	// Output plain text.
+	FormatText FormatType = "text"
+	// Output JSON.
+	FormatJSON FormatType = "json"
+)
```

Comment:
I think we also use `cbor` when generating transactions.

Reply:
Yeah, I noticed that, but it seems to be only used by that specific command, so it didn't make sense to me to have it as a global option if no other command supports it :)